AGE_65_PLUS.qmd

```diff
@@ -0,0 +1,126 @@
+```{r}
+#| warning: false
+#| message: false
+library(tidycensus)
+library(tidyverse)
+library(choroplethr)
+library(choroplethrMaps)
+
+df = get_acs(geography = "State",
+             variables = c(pop_65_plus = "S0101_C01_030"),
+             survey    = "acs1",
+             year      = 2021,
+             output    = "wide",
+             cache_table = TRUE)
+totaldf = get_acs(geography = "State",
```

Comment:
Please spell this using "snake case". I.e.e `total_df` instead of `totaldf`. The same goes for `percentdf` below.